.github/actions/fullnode-sync/action.yaml

```diff
@@ -32,6 +32,7 @@ runs:
       run: |
         echo "${HOME}/bin/" >> $GITHUB_PATH # default INSTALL_DIR to path
         scripts/dev_setup.sh -b # Install dependencies
+        sudo apt-get update -y && sudo apt-get install
```

Comment:
Is it missing the package name to install?

Reply:
Yeah, I saw a flurry of failures last night about being unable to find the packages, etc. But, this morning things look okay. This was the only change I made to the jobs yesterday, so wanted to see if rolling back would fix it, but I guess we're good. I'll close this out and monitor what happens... 😄